plugin/src/main/groovy/com/here/gradle/plugins/jobdsl/RestJobManagement.groovy

```diff
@@ -98,7 +98,8 @@ class RestJobManagement extends AbstractJobManagement implements DeferredJobMana
 
     @SuppressWarnings('ParameterCount')
     RestJobManagement(ItemFilter filter, boolean disablePluginChecks, boolean dryRun, String jenkinsUrl,
-                      String jenkinsUser, String jenkinsApiToken, String proxyUrl, Jenkins jenkins) {
+                      String jenkinsUser, String jenkinsApiToken, String proxyUrl, Boolean ignoreSSLErrors,
```

Comment:
Please use camel case for abbreveations in the whole commit, for consistency: `ignoreSslErrors`